core-plugins/src/main/java/io/cdap/plugin/batch/aggregator/DedupAggregator.java

```diff
@@ -205,7 +205,7 @@ private void validateSchema(Schema inputSchema, List<String> uniqueFields,
               field.getSchema().getNonNullable().getType() : field.getSchema().getType();
       if ((function.getFunction() == DedupConfig.Function.MAX || function.getFunction() == DedupConfig.Function.MIN)
         && !ALLOWED_SCHEMA_TYPES.contains(fieldType)) {
-        collector.addFailure(String.format("Unsupported filter operation %s(%s): Field '%s' is non numeric ",
+        collector.addFailure(String.format("Unsupported filter operation %s(%s): Field '%s' type is not allowed",
```

Comment:
nit: change to "Field <bla> has a type that is not supported for deduplication operations